src/powerquery-language-services/inspection/commonTypes.ts

```diff
@@ -19,4 +20,6 @@ export interface Inspected {
     readonly triedExpectedType: TriedExpectedType;
     readonly typeCache: TypeCache;
     readonly parseState: PQP.Parser.ParseState;
+
+    collectAllIdentifiersBeneath(entryNodeId: Ast.Identifier): Promise<Ast.Identifier[]>;
```

Comment:
Mark as `readonly`. Unless there's a specific reason I try to keep all properties that way.

Reply:
To make a readonly method try this syntax instead: `readonly collectAllIdentifiersBeneath: (entryNodeId: Ast.Identifier) => Promise<Ast.Identifier[]>;`